src/Common/recorder/ClipSortLogGenerator.h

```diff
@@ -17,6 +17,14 @@ class ClipSortLogGenerator : public JamMetadataWriter
     {
         return QCoreApplication::translate("Recorder::ClipSortLogGenerator", "Generate Reaper 'clipsort.log' file");
     }
+    inline void setJamDir(QString newJamName, QString recordBasePath) override {
```

Comment:
@pljones , a have just one little code review: too much code for a inline function here?